cypress/e2e/user-flows/onboarding.cy.js

```diff
@@ -0,0 +1,54 @@
+describe("onboarding", () => {
+  beforeEach(() => {
+    cy.setSliceMachineUserContext({
+      hasSeenTutorialsTooTip: false,
+      isOnboarded: false,
+    });
```

Comment:
I wonder if we should set the localStorage for these tests. Having an empty localStorage instead of `false` values could be more representative of a user who never logged in to SM. WDYT ?